canonical/ubuntu.com
# Rename default form template to a more generic name

## Done

- Rename all occurrences of `_cloud-contact-us-form.html` to `_default-contact-us-form.html`
- This is a duplicate PR of the existing work in https://github.com/canonical/ubuntu.com/pull/14107, rebasing the feature branch must've rolled back this work.

## QA

- Go to affected forms e.g https://ubuntu-com-14182.demos.haus/ai/contact-us
- See that form loads as expected

## Issue / Card

Fixes [WD-13646](https://warthogs.atlassian.net/browse/WD-13646)

## Screenshots

[If relevant, please include a screenshot.]


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)


[WD-13646]: https://warthogs.atlassian.net/browse/WD-13646?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
